mbs_results/selective_editing.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+
+
+def create_standardising_factor(
+    dataframe: pd.DataFrame,
+    reference: str,
+    period: str,
+    domain: str,
+    question_code: str,
+    predicted_value: str,
+    imputation_marker: str,
+    a_weight: str,
+    o_weight: str,
+    g_weight: str,
+    auxiliary_value: str,
+    previous_period: int,
```

Comment:
It looks like `reference`, `imputation_marker`, `auxiliary_value`  are not used for the standardised factors and the function just drops them, i think it would be better dropping them outside of this function